src/frontend/hooks/persistedState/usePersistedProjectId.ts

```diff
@@ -2,8 +2,8 @@ import {type StateCreator} from 'zustand';
 import {createPersistedState} from './createPersistedState';
 
 type ProjectIdSlice = {
-  projectId: string | undefined;
-  setProjectId: (id: string | undefined) => void;
+  projectId?: string;
```

Comment:
same comment about the TS optional field distinction. would lean towards the original definition because we know we instantiate the state with a `projectId: undefined` (see line 10 in this file)

```suggestion
  projectId: string | undefined;
```

Reply:
https://github.com/digidem/CoMapeo-mobile/pull/89/commits/1a9a0ef294e7d4226dc3159d35cf89c0a973022e